setup.py

```diff
@@ -23,12 +23,10 @@
     Generated by: https://openapi-generator.tech
 """
 
-from os import path
 from setuptools import setup, find_packages  # noqa: H301
 
 
-NAME = "apache-airflow-client"
-VERSION = "1.0.0"
+VERSION = "2.0.0"
```

Comment:
@kaxil I think using ver 2.0.0 isn't correct, as we've already released 2.0.0, so the code in the master is post 2.0.0 cut. We might name it 2.0.0.dev or whatever it is in airflow master.

Reply:
Yeah: let's change it to `2.1.0.dev`